toolchain/check/import_cpp.cpp

```diff
@@ -18,20 +18,57 @@
 #include "toolchain/diagnostics/format_providers.h"
 
 namespace Carbon::Check {
+namespace {
+
+auto GenerateCppIncludesHeaderFilename(llvm::StringRef importing_file_path)
+    -> std::string {
+  return llvm::Twine(importing_file_path)
+      .concat(".generated.cpp_imports.h")
+      .str();
+}
+
+auto GenerateCppIncludesHeaderCode(
```

Comment:
Can  you add a brief comment summarizing what this does? e.g.:

```suggestion
// Generates C++ file contents to #include all requested imports.
auto GenerateCppIncludesHeaderCode(
```

Style encourages putting comments on [almost all functions](https://google.github.io/styleguide/cppguide.html#Function_Comments).

Note, you might also consider a shorter function name -- maybe something like `GenerateCppIncludes`. At least "header" and "code" seem to have a very similar meaning.

Reply:
Done.